### ISSUE TITLE
I've added you to the awesome-streamlit resources list. Hope it's ok?

### ISSUE THREAD

**Issue by MarcSkovMadsen** [Jcharis/Streamlit_DataScience_Apps#1]
Hi Jesse

I've added you to the [awesome-streamlit.org](https://awesome-streamlit.org) resources list. 

Hope it's ok?

Feel free to add your example(s) to the gallery as code contributions to 
the [repo](https://github.com/marcskovmadsen/awesome-streamlit)

I might also do it later if I get the time. Hope that would be ok as well?

Marc

![image](https://user-images.githubusercontent.com/42288570/67260821-d1e50200-f49d-11e9-872f-9044d3aab92c.png)

**Comment by Jcharis** [comment]
Yes it is ok, Thanks a lot Marc ,I really appreciate it. Awesome Framework.
Great work
